Xamarin.PropertyEditing/Reflection/ReflectionEnumPropertyInfo.cs

```diff
@@ -65,7 +65,7 @@ public override void SetValue<TValue> (object target, TValue value)
 			}
 		}
 
-		public override TValue GetValue<TValue> (object target)
+		public override async Task<TValue> GetValueAsync<TValue> (object target)
```

Comment:
These methods are `async` but lack any `awaits`. Is that intentional? They'll run synchronously.